src/ape/managers/chain.py

```diff
@@ -680,7 +680,7 @@ def get_container(self, contract_type: ContractType) -> ContractContainer:
 
     def instance_at(
         self,
-        address: Union[str, "AddressType"],
+        address: AddressType,
```

Comment:
This one is actually right, because this can accept string values that convert to `AddressType`